coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
+    '''
+    Checks if there are any conflicting aruments passed
+
+    :return:            True if no conflicts
+    :raises SystemExit: If there are conflicting arguments (exit code: 2)
+    '''
+    conflicts = defaultdict(set)
+
+    # Add any conflict definitions here
+    conflicts["no_config"].add("save")
+    conflicts["no_config"].add("find_config")
+
+    args = list(conflicts.keys())
+    for arg in args:
+        for conflict in conflicts[arg]:
+            conflicts[conflict].add(arg)
+
+    for section in cli_sections:
+        for arg in cli_sections[section]:
+            for conflict in conflicts[arg]:
```

Comment:
I was talking with @sils1297 yesterday. Is this http://ix.io/rrS/py better? It's simpler and removes the redundant lines [105-109].